rosdep/kinetic.yaml

```diff
@@ -1,13 +1,37 @@
+boxer_control:
+  ubuntu: ros-kinetic-boxer-control
```

Comment:
Are these needed as deps since they are in the distribution?

Reply:
Might not need them anymore.  Still working through the boxer packages